src/components/InputMoney.js

```diff
@@ -64,83 +64,70 @@ class InputMoney extends Component {
     error: false
   };
 
-  componentDidMount() {
-    if (typeof this.props.getRef === "function") this.props.getRef(this.input);
-  }
-
   _onChange = value => {
     const formatedValue = formatAmount(value);
     if (this.props.balance) this.updateRemaining(value);
-    typeof this.props.onChange === "function" &&
-      this.props.onChange(formatedValue);
+    typeof this.props.onChange === "function" && this.props.onChange(formatedValue);
     if (value.length) this.setState({ hasValue: true });
     else this.setState({ hasValue: false });
     return formatedValue;
   };
 
-  _onBlur = value => {
+  _onBlur = elm => {
+    let valueObj = elm;
+    if (!elm.value.length) this.setState({ hasValue: false });
```

Comment:
There was similar func, that also included check `if (!elm) return null;` is it missing here?

Reply:
tbh the code `if (!elm) return null;` is actually redundant as the a value will always exits coming into the `_onBlur` function either natively or manually

Reply:
the purpose of the code was to set the `valueObj = elm` and have it go through validation check and mutate `valueObj` accordingly else return its initial assigned value